CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## 0.17.11 (Unreleased)
 
+- Add detection of CI environment when running inside Azure Pipelines.
```

Comment:
Nit: Perhaps "Add support for Azure Pipelines in CI/CD environment detection"? Just to use a more active voice?

Reply:
Done.